packages/neo-one-smart-contract-lib/src/__data__/contracts/lifecycle/TestPausable.ts

```diff
@@ -0,0 +1,32 @@
+import { Address, SmartContract, constant } from '@neo-one/smart-contract';
+/* tslint:disable-next-line: no-implicit-dependencies */
```

Comment:
Is there a way we can add a *solid* TSLint exclude for these files? Seeing this in every `__data__` file is a bit unintuitive. Not sure if we have that control for single rule ignoring across a glob.

Reply:
@danwbyrne totally agree, but it looks like they don't want to include a Conditional configuration [https://github.com/palantir/tslint/issues/3447]